access_point/wifi_test.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+WIFI=''
+
+if nc -zw1 google.com 443
+then 
+    echo "connection successful"
+    WIFI='1'
+elif ping -q -t 5 -w1 -c1 8.8.8.8 t
+then
+    echo "connection successful"
+    WIFI='1'
+
+else
+    echo "wifi not connected"
+    WIFI='0'
```

Comment:
@stealthanthrax how did you plan to check for 3 times password failure you discussed here https://github.com/fossasia/susi_linux/issues/146.

I suppose this only checks the connection establishment, right?

Reply:
@prateekiiest i think he meant check 3 times to ensure that there's is a connection else password is wrong

Reply:
but then again the code doesn't does what I just specified. 
@stealthanthrax either update your code or update your issue pls. Thanks

Reply:
@cweitat , i'll be updating the code